_includes/master/reqs-sys.md

```diff
@@ -41,12 +41,12 @@ Ensure that your hosts and firewalls allow the following traffic.
 | flannel networking (VXLAN)                                   | All                 | Bidirectional   | UDP 4789 |
 | etcd datastore                                               | etcd hosts          | Incoming        | [Officially](http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt) TCP 2379 but can vary |
 | {{site.prodname}} networking, cluster of more than 50 nodes  | Typha agent hosts   | Incoming        | TCP 5473 (default) |
-| All                                                          | kube-apiserver host | Incoming        | Often TCP 443 or 6443: check the `port` value returned by `kubectl get svc kubernetes -o yaml` |
+| All                                                          | kube-apiserver host | Incoming        | Often TCP 443 or 6443: check the `targetPort` value returned by `kubectl get svc kubernetes -o yaml` |
```

Comment:
Like I've said in person a few times... I think this approach of asking the user to check kubectl is awkward. It's not really the source of truth and in order to use kubectl you need to know what the port is already.

We should really say "The Kubernetes secure port. Typically TCP port 443"